src/styles/tickets/frontend.pcss

```diff
@@ -3,308 +3,339 @@
  *
  * @since  0.3.0-alpha
  */
-.tribe-block__tickets {
+.tribe-common.tribe-block__tickets {
+	border: 1px solid var(--color-border-secondary);
+	border-radius: 4px;
```

Comment:
there is a border radius variable we can use here.

Reply:
Yeah, these are going away - ignore this file.